test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const path = 'https://api.github.com';
+
+let response;
+
+describe('Github user list', () => {
```

Comment:
Missing first test, which is getting all the users that are obtained by deafult (That means, without query parameters)